modules/social_features/social_group/social_group.install

```diff
@@ -716,19 +716,13 @@ function social_group_update_8905() {
  */
 function social_group_update_8906() {
   $config_factory = \Drupal::configFactory();
-
-  $group_types = [
-    'public_group',
-    'open_group',
-    'closed_group',
-  ];
-
-  foreach ($group_types as $group_type) {
-    $config = $config_factory->getEditable('group.type.' . $group_type);
+  $group_types = \Drupal::entityTypeManager()->getStorage('group_type')->loadMultiple();
```

Comment:
Just for reviewers understanding:

At the time of Open Social 8.x, we only shipped with public, closed, and open group types. That is why we never considered writing the update hook for "other" group types as well. We only wanted to focus on default groups shipped by Open Social.